plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/MavenPathParser.java

```diff
@@ -10,7 +10,7 @@
  * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
  * Eclipse Foundation. All other trademarks are the property of their respective owners.
  */
-package org.sonatype.nexus.repository.maven.internal;
+package org.sonatype.nexus.repository.maven;
```

Comment:
ITs run in exactly the same environment as the distribution, with the test code in its own bundle. If you need to access internals for some reason then you need to take advantage of the `GlobalComponentLookupHelper` and write your IT in groovy, as this gives you access to internal types and handles a lot of the necessary reflection for you. However I'd argue a test that needs to delve into internals is not really an IT but more of a UT.

All that said, moving this component out of internal (or exposing it via a public API) is the right thing to do if it is meant to be re-used elsewhere.